WootricSDK/WootricSDK/WTRSettings.h

```diff
@@ -65,6 +70,12 @@
 - (NSString *)socialShareQuestionText;
 - (NSString *)socialShareDeclineText;
 
+- (UIColor *)sendButtonBackgroundColor;
+- (UIColor *)noThanksColor;
+- (UIColor *)sliderColor;
+- (UIColor *)thankYouButtonBackgroundColor;
+- (UIColor *)socialSharingColor;
+
```

Comment:
I guess you're declaring this methods because you're going to overwrite each one, but you don't need to declare them, just declare and implement them in the .m file (just like I saw you did below)

When you declare a @property you get the gettters and setters methods already.